bin/test_validator

```diff
@@ -38,13 +38,12 @@ pubber/bin/build
 
 bin/reset_config $site_path $project_id $device_id
 
-bin/validator $site_path $project_id &
+echo Starting validator, output in $VALIDATOR_OUT
+bin/validator $site_path $project_id > $VALIDATOR_OUT 2>&1 &
 vpid=$!
-sleep 10
 echo Started validator pid $vpid
+sleep 10
```

Comment:
Is this to make sure validator is running? Can we do something more deterministic than wait?

Reply:
There is a deterministic check right after this, this was just to keep the unnecessary logging at a minimum... I added a comment explaining.